barretenberg/cpp/src/barretenberg/vm/avm/tests/mem_opcodes.test.cpp

```diff
@@ -164,10 +164,10 @@ class AvmMemOpcodeTests : public ::testing::Test {
         if (indirect) {
             auto const& mem_ind_a_row = trace.at(mem_ind_a_row_idx);
             EXPECT_THAT(mem_ind_a_row,
-                        AllOf(MEM_ROW_FIELD_EQ(tag_err, 0),
+                        AllOf(MEM_ROW_FIELD_EQ(tag_err, indirect_uninitialized ? 1 : 0),
                               MEM_ROW_FIELD_EQ(r_in_tag, static_cast<uint32_t>(AvmMemoryTag::U32)),
                               MEM_ROW_FIELD_EQ(tag,
-                                               indirect_uninitialized ? static_cast<uint32_t>(AvmMemoryTag::U0)
+                                               indirect_uninitialized ? static_cast<uint32_t>(AvmMemoryTag::FF)
                                                                       : static_cast<uint32_t>(AvmMemoryTag::U32)),
                               MEM_ROW_FIELD_EQ(addr, src_offset),
```

Comment:
`tag_err` if you try to do an indirect read and the address read from memory is a field and not a u32